server/validation/validator.js

```diff
@@ -3,9 +3,32 @@ import Joi from 'joi';
 export default {
   login: {
     body: {
-      email: Joi.when('username', { is: Joi.exist(), then: Joi.string().email(), otherwise: Joi.string().email().required() }),
-      username: Joi.string().alphanum().min(3).max(30),
+      email: Joi.when('username', {
+        is: Joi.exist(),
+        then: Joi.string().email(),
+        otherwise: Joi.string()
+          .email()
+          .required(),
+      }),
+      username: Joi.string()
+        .alphanum()
+        .min(3)
+        .max(30),
       password: Joi.string().required(),
     },
   },
+  research: {
+    body: {
+      title: Joi.string().required(),
+      files: {
+        researchFile: Joi.object({
+          fieldname: Joi.string().required(),
+          mimetype: Joi.string()
+            .valid('application/pdf')
+            .error(() => 'Research file must be a pdf document'),
+          filename: Joi.string().required(),
+        }).required(),
```

Comment:
We could modify the error here to a more user friendly message as opposed to the current researchFile is required.
```suggestion
        }).required()
        .error(() => 'Select a research file for upload')
```